starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+body{
+  font-family: 'Open Sans', sans-serif;
+  font-size: 1em;
```

Comment:
Cuidado con 1em en el body, la unidad em suele necesitar una referencia para poder escalarse. Si me sabéis justificar por qué lo habéis usado os lo acepto, pero hasta entonces os paso un artículo que lo explica super bien! ;) [https://www.sitepoint.com/power-em-units-css/](url)